cartography/intel/github/repos.py

```diff
@@ -785,6 +929,69 @@ def load_collaborators(
         )
 
 
+@timeit
+def load_python_requirements(
```

Comment:
Nit: the diff in this PR confused me. I thought that this was a whole new function so I was confused about why we were adding in new manually written queries. If possible, avoid moving existing functions around too much so that it doesn't mess up the diffs and makes them easier to review